notes/fidelity.html

```diff
@@ -12,12 +12,7 @@ <h2 class="ui header">
         For <em>authentication</em>, Fidelity uses either a physical token provided by Fidelity
         or the Symantec VIP Access smartphone app.
         <br>
-        Fidelity only offers 2FA on certain account types. For example, they
-        do not support it for institutional retirement accounts of the kind
-        that might be set up through an employer.
-        <br>
-        It appears there is no information about this service provided
-        online and it must be enabled and managed by phone.
+        Fidelity may only offer 2FA on certain account types.
     </p>
 
     <p>
```

Comment:
I believe everything below here about "As of October 2016..." and "That this is different than the weaker Transaction Security..." is outdated. The two links below still link to pages that exist, but the page content has changed and the quoted text is no longer there, and Fidelity is no longer advertising Transaction Security as a feature, instead advertising 2FA at login. I'd recommend replacing all this outdated text with a simple link to Fidelity's newer page documenting 2FA: https://www.fidelity.com/security/extra-security-login